contoso_supermarket/operations/freezer_monitoring/values.yaml

```diff
@@ -0,0 +1,32 @@
+broker:
+  mqttBroker:
+    image:
+      repository: jumpstartagora.azurecr.io/contoso-supermarket/mqtt-broker
```

Comment:
we should add a placeholder for acr like _acrName_ and add it in the github action "update-files" to replace it

Reply:
looking into this...

Reply:
These ones are not changing. They will always pull from jumpstartagora.azurecr.io